tests/sqlparser_snowflake.rs

```diff
@@ -4232,3 +4232,122 @@ fn test_snowflake_create_view_with_composite_policy_name() {
         r#"CREATE VIEW X (COL WITH MASKING POLICY foo.bar.baz) AS SELECT * FROM Y"#;
     snowflake().verified_stmt(create_view_with_tag);
 }
+
+#[test]
```

Comment:
Could we add a test scenario for how this interacts with the [double dot](https://github.com/apache/datafusion-sqlparser-rs/blob/018992b22a78586dc4f176e80c6f1bf171bbfd7d/tests/sqlparser_snowflake.rs#L3330) feature?

Reply:
Yeah was thinking a test as in your example to demonstrate if/how the parser handles it. If its unsupported then it'll be a negative test